api/CSSImportRule.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "CSSImportRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSImportRule",
```

Comment:
This is spec'd in https://drafts.csswg.org/cssom/#the-cssimportrule-interface, and that matches the subfeatures being added here.